scripts/murmur.ini

```diff
@@ -274,6 +367,13 @@ allowping=true
 ; (Note that you should only change this value if you know what you are doing)
 ;kdfIterations=-1
 
+; TODO these needs documentation:
+;autobanAttempts=10
+;autobanTimeframe=120
+;autobanTime=300
+;loggroupchanges=false
+;logaclchanges=false
```

Comment:
See https://wiki.mumble.info/wiki/Murmur.ini#Logging_.26_Debugging

Reply:
Weird enough, the whole"== Logging & Debugging ==" section is not visible for me in the wiki page (or its html source), but it's present in "view source" for the wiki page!

Done.